Src/FluentAssertions/Formatting/Formatter.cs

```diff
@@ -115,7 +113,7 @@ private static string FormatChild(string path, object childValue, bool useLineBr
                 {
                     return $"{{Cyclic reference to type {childValue.GetType()} detected}}";
                 }
-                else if (graph.Depth > MaxDepth)
+                else if (graph.Depth > 5)
```

Comment:
Maybe make this configurable instead of constant?

Reply:
Personally I don't see why anybody would ever see an object graph in their unit test output window that goes beyond 5 levels deep.

Reply:
Maybe for comparing json or xml objects?

Reply:
For that we have specialized APIs that don't require cyclic reference detection (which is the real cause of the problem mentioned here)